### PR TITLE
Fix typos in GammaRV and HalfCauchyRV

### PR DESCRIPTION
**Pull request by ricardoV94**

### CODE REVIEW

File: aesara/tensor/random/basic.py [hunk 1]
```diff
@@ -73,18 +73,18 @@ def rng_fn(cls, rng, loc, scale, size):
...
     _print_name = ("Gamma", "\\operatorname{Gamma}")
 
     def __call__(self, shape, rate, size=None, **kwargs):
-        return super().__call__(shape, 1.0 / rate, size=size, **kwargs)
+        return super().__call__(shape, rate, size=size, **kwargs)
 
     @classmethod
-    def rng_fn(cls, rng, shape, scale, size):
-        return stats.gamma.rvs(shape, scale=scale, size=size, random_state=rng)
+    def rng_fn(cls, rng, shape, rate, size):
```
Comment (ricardoV94) [inline aesara/tensor/random/basic.py:86]: Not sure about this, I though it makes sense for the `__call__` and `rng_fn` to treat the same inputs equally, but I actually don't know what is the purpose of `__call__` here. Happy to revert.

File: aesara/tensor/random/basic.py [hunk 2]
```diff
@@ -73,18 +73,18 @@ def rng_fn(cls, rng, loc, scale, size):
...
-    name = "halfnormal"
+    name = "gamma"
     ndim_supp = 0
     ndims_params = [0, 0]
     dtype = "floatX"
     _print_name = ("Gamma", "\\operatorname{Gamma}")
 
     def __call__(self, shape, rate, size=None, **kwargs):
-        return super().__call__(shape, 1.0 / rate, size=size, **kwargs)
+        return super().__call__(shape, rate, size=size, **kwargs)
```
Comment (brandonwillard) [inline aesara/tensor/random/basic.py:83]: Why this breaking change?

**Review by brandonwillard — CHANGES_REQUESTED** [review]

File: aesara/tensor/random/basic.py (hunk 2 above)
Comment (ricardoV94) [reply aesara/tensor/random/basic.py:83]: No good reason, just wanted to check if it broke things. I thought the same variable names rate / scale should be used between `__call__` and `rng_fn` and I changed it here. Will revert

**Comment by ricardoV94** [comment]
Just the typos now

**Review by brandonwillard — APPROVED** [review]

**Pull request merged**